### ISSUE TITLE
Full Keyboard accessibility

### ISSUE THREAD

**Issue by Greyhawk62** [googlecodelabs/angular-accessibility#4]
This was a great tutorial to get me started with accessibility. One thing I was trying to do was to completely navigate the sight with just using the keyboard with text to speech turned on. When displaying the navigation page "Our Story" I could not get into the page content so the screen reader would read the page. I am thinking the way this should work is by clicking the navigation menu item (using the enter key) the focus should go into the page content. Then I'm not sure how I would go from the page content back to the main navigation. Since I am new to focusing on Accessibility and just using the keyboard I might be missing something.

Any thoughts would be appreciated.
